lib/dialects/sqlite3/schema/ddl.js

```diff
@@ -19,6 +19,11 @@ const {
   chunk,
 } = require('lodash');
 
+const ReuseConnection = (connection) =>
+  async function(next) {
+    return next(connection);
+  };
+
```

Comment:
FYI: This function is currently duplicated in a few places.  We can consolidate it as part of the re-implementation.